src/helpers/helpers.canvas.js

```diff
@@ -15,6 +15,23 @@ const TWO_THIRDS_PI = PI * 2 / 3;
  * @namespace Chart.helpers.canvas
  */
 
+/**
+ * Converts the given font object into a CSS font string.
+ * @param {object} font - A font object.
+ * @return {string|null} The CSS font string. See https://developer.mozilla.org/en-US/docs/Web/CSS/font
+ * @private
+ */
+export function toFontString(font) {
+	if (!font || isNullOrUndef(font.size) || isNullOrUndef(font.family)) {
+		return null;
+	}
+
+	return (font.style ? font.style + ' ' : '')
```

Comment:
Not sure if we want to make a call now on template strings in our code case but an option here

Reply:
We'd have to setup a babel plugin to support IE11. Probably easier to just leave it as is